### PR TITLE
Persian translations added

### PR DESCRIPTION
**Pull request by arashveysi3**
Added Persian translations fully according to Contriibution.MD instructions. 
plus I updated Readme.MD (Persian included in supported languages)
open to future contributions and thanks for your great application. 💯 
Hope it helps!

### CODE REVIEW

File: src/utils/translations/fa.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
+const fa = {
+  // Home Tab
+  Home: "خانه",
+  "Period in": "تا پریود بعدی",
+  "no info": "بدون اطلاعات",
+  Period: "پریود",
+  today: "امروز",
+  Delay: "تاخیر",
+  Days: "(0)[روز];(1)[روز];(2-4)[روز];(5-20)[روز];(21)[روز];(22-24)[روز];(25-30)[روز];(31)[روز];(32-34)[روز];(35-40)[روز];",
```
Comment (imblowfish) [inline src/utils/translations/fa.ts:9]: You dont need to write all these options in this language, instead you can use:

```ts
  Days: "(0-inf)[روز]",
```

**Review by imblowfish — CHANGES_REQUESTED** [review]

**Comment by imblowfish** [comment]
Hello @arashveysi3. Thank you for your contribution

I have a question. Am I right that Persian alphabet has right-to-left direction and right text alignment?

If yes, I have a request to you to run oeri locally on your laptop and show us problems in current UI which should be fixed according to right-to-left alphabet

Of course if you are a native speaker and this is will not be difficult for you

You just need to exexute `npm start` and attaxh screenshot with problems

To mark problems in the app you can use any image editor. I can recommend excalidraw.com, for example

**Comment by arashveysi3** [comment]
Hey @imblowfish Thanks for reviewing my PR.
Yes, you are right in it supposed to be RTL instead of LTR.
Good news is with a little of your guide I can fix it! 

the problem exists in all of the pages but as an example I took a screenshot of the main page
as you suggested to me, I used https://excalidraw.com and what a great app!

https://excalidraw.com/#room=cc4fb9147c008706c73e,GJ9evYHPhNV-zwVIkDJDiw

File: src/utils/translations/fa.ts (hunk 1 above)
Comment (arashveysi3) [reply src/utils/translations/fa.ts]: Done boss!

**Comment by imblowfish** [comment]
> Hey @imblowfish Thanks for reviewing my PR.
> Yes, you are right in it supposed to be RTL instead of LTR.
> Good news is with a little of your guide I can fix it! 
> 
> the problem exists in all of the pages but as an example I took a screenshot of the main page
> as you suggested to me, I used https://excalidraw.com and what a great app!
> 
> https://excalidraw.com/#room=cc4fb9147c008706c73e,GJ9evYHPhNV-zwVIkDJDiw

Thank you!

We'll create additional ticket for rtl interface support. Until this, could you please move Persian language to the Beta languages? We'll return it back later

File: src/utils/translation.ts [hunk 2]
```diff
@@ -27,6 +28,7 @@ export const supportedLanguages = new Map<string, string>([
   ["fr", "français"],
   ["hi", "हिन्दी"],
   ["ru", "русский"],
+  ["fa", "پارسی"],
```
Comment (imblowfish) [inline src/utils/translation.ts:31]: We doesn't support rtl languages yet. Could you please move this language to the Beta section?

**Review by imblowfish — CHANGES_REQUESTED** [review]

File: src/utils/translation.ts (hunk 2 above)
Comment (arashveysi3) [reply src/utils/translation.ts]: Consider it done!

**Comment by arashveysi3** [comment]
> > Hey @imblowfish Thanks for reviewing my PR.
> > Yes, you are right in it supposed to be RTL instead of LTR.
> > Good news is with a little of your guide I can fix it!
> > the problem exists in all of the pages but as an example I took a screenshot of the main page
> > as you suggested to me, I used https://excalidraw.com and what a great app!
> > https://excalidraw.com/#room=cc4fb9147c008706c73e,GJ9evYHPhNV-zwVIkDJDiw
> 
> Thank you!
> 
> We'll create additional ticket for rtl interface support. Until this, could you please move Persian language to the Beta languages? We'll return it back later
> 
> And we could probably need your help. Can we tag you to check the fixes in the future?

Yes why not! Persian language moved to Beta language section for now. let's say mission accomplished! 
I'm open to any further contributions on this project happily. meanwhile I am fluent in Turkish too. and It's a LTR language so I will start to translate app to Turkish too.

**Review by imblowfish — APPROVED** [review]

**Comment by imblowfish** [comment]
@IraSoro review plz

**Review by IraSoro — APPROVED** [review]

**Comment by IraSoro** [comment]
@arashveysi3, thank you for your interest in the project and for the work you've done! Congratulations on your first contribution to `Peri`!

**Pull request merged**